Makefile

```diff
@@ -0,0 +1,91 @@
+# linter modules to be included/excluded
+LINT_INCLUDE=-E bodyclose,gocritic,gofmt,gosec,govet,nestif,nlreturn,revive,rowserrcheck
+LINT_EXCLUDE=-e G401,G501,G107
+
+help:
+	@echo 'Welcome!'
+	@echo ''
+	@echo 'This Makefile is designed to make the development work go smoothly.'
+	@echo 'In-depth description of how to use this Makefile can be found in the README.md'
+
+bootstrap: go-version-check
+		@for dir in sda sda-auth sda-download; do \
+			cd $$dir; \
+			go get ./...; \
+			cd ..; \
+		done
+		if ! command -v curl >/dev/null; then
+			echo "Can't install golangci-lint because curl is missing."
+			exit 1
+		fi
```

Comment:
maybe this needs to be a one-liner ? as I am getting
```
if ! command -v curl >/dev/null; then
/bin/sh: 1: Syntax error: end of file unexpected (expecting "fi")
make: *** [Makefile:13: bootstrap] Error 2
```

Reply:
🤦🏻 to early in the morning it seems.